core/src/main/java/org/akaza/openclinica/domain/xform/XformParserHelper.java

```diff
@@ -162,4 +162,21 @@ public void iterateUserControlsInBodyAndAddCommentElement(Document doc, Node bod
         return set;
     }
 
+    public List<String> bodyGroupNodePaths(Node outerNode, List<String> list) {
+        int outerNodeLength = outerNode.getChildNodes().getLength();
+        for (int b = 0; b < outerNodeLength; b++) {
+            Node node = outerNode.getChildNodes().item(b);
+            if (node instanceof Element && !node.getNodeName().equals("formhub") && !node.getNodeName().equals("meta")) {
```

Comment:
Can you break these conditions on separate lines for better readability?
if (node instanceof Element 
            && !node.getNodeName().equals("formhub") 
            && !node.getNodeName().equals("meta")) {